sites/ademe/src/backend/ademe/settings.py

```diff
@@ -0,0 +1,755 @@
+                "loaders": [
+                    "django.template.loaders.filesystem.Loader",
+                    "django.template.loaders.app_directories.Loader",
+                ],
+            },
+        }
+    ]
+
+    # Placeholders
+    CMS_PLACEHOLDER_CONF_OVERRIDES = {
+        "courses/cms/course_detail.html course_teaser": {
+            "name": _("Teaser"),
+            "plugins": ["LTIConsumerPlugin", "VideoPlayerPlugin"],
+            "limits": {
+                "LTIConsumerPlugin": 1,
+                "VideoPlayerPlugin": 1,
+            },
+        },
+    }
+
+    MIDDLEWARE = (
+        "richie.apps.core.cache.LimitBrowserCacheTTLHeaders",
+        "cms.middleware.utils.ApphookReloadMiddleware",
+        "django.middleware.security.SecurityMiddleware",
+        "django.contrib.sessions.middleware.SessionMiddleware",
+        "django.middleware.csrf.CsrfViewMiddleware",
+        "django.contrib.auth.middleware.AuthenticationMiddleware",
+        "django.contrib.messages.middleware.MessageMiddleware",
+        "django.middleware.locale.LocaleMiddleware",
+        "django.middleware.common.CommonMiddleware",
+        "django.middleware.clickjacking.XFrameOptionsMiddleware",
+        "dockerflow.django.middleware.DockerflowMiddleware",
+        "cms.middleware.user.CurrentUserMiddleware",
+        "cms.middleware.page.CurrentPageMiddleware",
+        "cms.middleware.toolbar.ToolbarMiddleware",
+        "cms.middleware.language.LanguageCookieMiddleware",
+        "dj_pagination.middleware.PaginationMiddleware",
+    )
+
+    INSTALLED_APPS = (
+        # ADEME stuff
+        "base",
+        # Richie stuff
+        "richie.apps.demo",
+        "richie.apps.search",
+        "richie.apps.courses",
+        "richie.apps.core",
+        "richie.plugins.glimpse",
+        "richie.plugins.html_sitemap",
+        "richie.plugins.large_banner",
+        "richie.plugins.lti_consumer",
+        "richie.plugins.nesteditem",
+        "richie.plugins.plain_text",
+        "richie.plugins.section",
+        "richie.plugins.simple_picture",
+        "richie.plugins.simple_text_ckeditor",
+        "richie",
+        # Third party apps
+        "dj_pagination",
+        "dockerflow.django",
+        "parler",
+        "rest_framework",
+        "storages",
+        # Django-cms
+        "djangocms_admin_style",
+        "djangocms_googlemap",
+        "djangocms_link",
+        "djangocms_picture",
+        "djangocms_text_ckeditor",
+        "djangocms_video",
+        "cms",
+        "menus",
+        "sekizai",
+        "treebeard",
+        "filer",
+        "easy_thumbnails",
+        # Django
+        "django.contrib.auth",
+        "django.contrib.contenttypes",
+        "django.contrib.sessions",
+        "django.contrib.admin",
+        "django.contrib.sites",
+        "django.contrib.sitemaps",
+        "django.contrib.staticfiles",
+        "django.contrib.messages",
+    )
+
+    # Languages
+    # - Django
+    LANGUAGE_CODE = "en"
```

Comment:
It should be `fr` by default like on `funmooc` for example.
```suggestion
    LANGUAGE_CODE = "fr"
```